File: Data Structures & Algorithms/products-of-array-discluding-self/submission-3.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int> ans;
        for(int i =0 ; i< nums.size() ;i++){
            int val = 1;
            for(int j =0 ; j< nums.size();j++){
                if(i != j){
                    val *= nums[j];
                }
            }
            ans.push_back(val);
        }
        return ans;
}
};

File: Data Structures & Algorithms/frequency-of-the-most-frequent-element/submission-3.java

class Solution {
    public int maxFrequency(int[] nums, int k) {
        Arrays.sort(nums);
        
        int l = 0, r = 0, ans = 1;
        long diff = 0;
        
        while (r < nums.length) {
            
            if (r > 0) {
                diff += (long)(nums[r] - nums[r - 1]) * (r - l);
            }
            
            while (diff > k) {
                diff -= (nums[r] - nums[l]);
                l += 1;
            }

            ans = Math.max(ans, r - l + 1);
            r += 1;
        }
        
        return ans;
    }
}